doc/source/architecture.rst

```diff
@@ -68,13 +65,13 @@ PBM Control Collections
 
 The config and state (current and historical) for backups is stored in
 collections in the MongoDB cluster or non-sharded replica set itself. These are
-put in the system ``admin`` db to keep them cleanly separated from user db
-namespaces. In a cluster only in the ``admin`` db on the ``configsvr``
-replicaset, not the shards.
+put in the system ``admin`` db of the config server replica set to keep them
```

Comment:
It is better style to use abbreviations sparingly: replace `db' with `database'.